Assets/Scripts/Game/Guilds/KnightlyOrder.cs

```diff
@@ -142,13 +141,13 @@ public override TextFile.Token[] TokensPromotion(int newRank)
             return DaggerfallUnity.Instance.TextProvider.GetRSCTokens(GetPromotionMsgId(newRank));
         }
 
-        private int GetPromotionMsgId(int rank)
+        private int GetPromotionMsgId(int newRank)
```

Comment:
Why does this need changing? This method doesn't care if rank is a new one or not, hence why I named the parameter just rank.

Reply:
Oh you mean it is the same name as an internal member? Yes, that's true so I understand now.

Reply:
Well I'm fairly comfortable hiding members in local scope, since you can always do this.rank to access the member. I think the rename is fine though.